python/lsst/pipe/tasks/imageDifference.py

```diff
@@ -191,6 +200,7 @@ def setDefaults(self):
         self.subtract['al'].kernel.active.spatialKernelOrder = 1
         self.subtract['al'].kernel.active.spatialBgOrder = 2
         self.doPreConvolve = False
+        self.doScaleTemplateVariance = False
```

Comment:
Remove this. It's redundant, with the original default you specified above. 

In fact, every default in this set defaults method that is NOT a subtask config should be moved above.

Reply:
These actually override the default values, I can't see why.